app/models/api_token.rb

```diff
@@ -0,0 +1,11 @@
+require 'securerandom'
+
+class ApiToken < ActiveRecord::Base
```

Comment:
Do we really want to pull in this class and all of the changes around this?  I think not, but I could be wrong.